### DISCUSSION TITLE
IP ADAPTER support?

### DISCUSSION THREAD

**Discussion by pellaaa93** [continue-revolution/sd-webui-animatediff#335] [General]
Hello is it possible to use ip-adapter (face, plus, etc) for init images with the a1111 extension? I am not sure if is not supported or I am doing something wrong but can't get any consistence between init image and output video.

**Comment by Li1177** [comment]
The IP ADAPTER can be used in Animatediff. I've been using this plugin recently, and it almost serves as a replacement for Lora.

**Comment by continue-revolution** [comment]
All ip adapters should have been usable now
